reference/docs-conceptual/whats-new/What-s-New-in-PowerShell-Core-62.md

```diff
@@ -6,43 +6,136 @@ ms.date: 03/28/2019
+```
+
+```Output
+Get-Commnd : The term 'Get-Commnd' is not recognized as the name of a cmdlet, function, script file, or operable program.
+Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
+At line:1 char:1
++ Get-Commnd
++ ~~~~~~~~~~
++ CategoryInfo          : ObjectNotFound: (Get-Commnd:String) [], CommandNotFoundException
++ FullyQualifiedErrorId : CommandNotFoundException
+
+
+Suggestion [4,General]: The most similar commands are: Get-Command, Get-Content, Get-Job, Get-Module, Get-Event, Get-Host, Get-Member, Get-Item, Set-Content.
+```
+
+### Implicit Remoting Batching
+
+When using [implicit remoting](https://devblogs.microsoft.com/scripting/remoting-the-implicit-way/)
+in a pipeline, PowerShell treats each command in the pipeline independently. Objects are repeatedly
+serialized and de-serialized between the client and remote system over the execution of the
+pipeline.
+
+With this feature, PowerShell analyzes the pipeline to determine if the command is safe to run and
+it exists on the target system. When true, PowerShell executes the entire pipeline remotely and only
+serializes and de-serializes the results back to the client.
+
+```powershell
+Enable-ExperimentalFeature -Name PSImplicitRemotingBatching
+```
+
+A real-world test of `Get-Process | Sort-Object` over localhost decreases from 10-15 seconds to
+20-30 **milliseconds**. The feature only needs to be enabled on the client. No changes are required
+on the server.
+
+### Temp Drive
+
+```powershell
+Enable-ExperimentalFeature -Name PSTempDrive
+```
+
+If you're using PowerShell Core on different operating systems, you'll discover that the environment
+variable for finding the temporary directory is different on Windows, macOS, and Linux! With this
+feature, you get a [PSDrive][] called `Temp:` that is automatically mapped to the temporary
+folder for the operating system you are using.
+
+#### Example
+
+```powershell
+PS> "Hello World!" > Temp:/hello.txt
+PS> Get-Content Temp:/hello.txt
+Hello World!
+```
+
+Be aware that native file commands (like `ls` on Linux) are not aware of PSDrives and won't see this
+`Temp:` drive.
+
+### Abbreviation Expansion
+
+PowerShell cmdlets are expected to have descriptive nouns. This results in long names that are more
+difficult to type. This feature allows you to just type the uppercase characters of the cmdlet and
+use tab-completion to find a match.
+
+```powershell
+Enable-ExperimentalFeature -Name PSUseAbbreviationExpansion
+```
+
+#### Example
+
+```powershell
+PS> i-arsavsf
+```
+
+If you hit tab, and have the Azure PowerShell [Az](https://www.powershellgallery.com/packages/Az)
+module installed, it will autocomplete to:
+
+```Output
+PS> Import-AzRecoveryServicesAsrVaultSettingsFile
+```
+
+> [!NOTE]
+> This feature is intended to be used interactively. Abbreviated forms of cmdlets can't be executed.
+> This feature is not a replacement for aliases.
 
 ## Breaking Changes
```

Comment:
There should be more of these if you continue scrolling through the changelog for previews. E.g.
> Make Join-String -InputObject 1,2,3 result equal to 1,2,3 | Join-String result (#8611) (Thanks @sethvs!)

is in 6.2.0-rc. And there's 4 in preview.4. 2 in preview.3, 4 in preview.2, and 1 in preview.1.

Did you happen to look at those for the rest of the content?